R/database.R

```diff
@@ -1248,22 +1248,17 @@ get_manta_sv = function(min_vaf = 0.1,
                         from_flatfile = TRUE,
                         projection = "grch37"){
 
-  if(projection != "grch37"){
-    message("Currently, only grch37 is supported")
-    return()
-  }
-
   db = config::get("database_name")
   table_name = config::get("results_tables")$sv
-    if(!missing(region)){
-      region = gsub(",", "", region)
-      #format is chr6:37060224-37151701
-      split_chunks = unlist(strsplit(region, ":"))
-      chromosome = split_chunks[1]
-      startend = unlist(strsplit(split_chunks[2], "-"))
-      qstart = startend[1]
-      qend = startend[2]
-    }
+  if(!missing(region)){
+    region = gsub(",", "", region)
+    #format is chr6:37060224-37151701
+    split_chunks = unlist(strsplit(region, ":"))
+    chromosome = split_chunks[1]
+    startend = unlist(strsplit(split_chunks[2], "-"))
+    qstart = startend[1]
+    qend = startend[2]
+  }
 
   #this table stores chromosomes with un-prefixed names. Convert to prefixed chromosome if necessary
```

Comment:
I'm realizing now why the remote functionality isn't working. The bundled snakefile currently doesn't sync the merged manta file locally. I think this is because I was assuming get_combined_sv would entirely replace get_manta_sv

Reply:
Got it, let's revisit this.